src/exitmap.py

```diff
@@ -322,17 +324,18 @@ def select_exits(args, module):
 
     exit_destinations = relayselector.get_exits(
         args.tor_dir,
-        good_exit       = args.all_exits or (not args.bad_exits),
```

Comment:
I think I prefer the previous version. This was also discussed here: https://github.com/NullHypothesis/exitmap/pull/48#issuecomment-206876969